ansible/tasks/initdb.yml

```diff
@@ -61,4 +92,17 @@
     password: "{{ db_password }}"
     force_basic_auth: yes
   with_items: "{{ db.authkeys }}"
-  when: result.status == 404
+  when: dbexists is defined and dbexists.status == 404
+
+- name: secure db against anonymous access
```

Comment:
This is additive. The role is currently creating the db unsecured. I wanted the option to toggle secure vs insecure mode. That strikes me more as a task than a deployment role. Additionally, to put this in the role and toggle it we would need a different flag - since mode is used for up/down of the container.